worlds/v6/docs/setup_en.md

```diff
@@ -13,10 +13,14 @@
 
 # Joining a MultiWorld Game
 
-To join, set the following launch options: `-v6ap_name YourName -v6ap_ip ServerIP:Port`.
-Optionally, add `-v6ap_passwd "YourPassword"` if the room you are using requires a password. All parameters without quotation marks.
-The Name in this case is the one specified in your generated .yaml file.
-In case you are using the Archipelago Website, the IP should be `archipelago.gg`.
+To join an Archipelago MultiWorld game, you must set the game's launch options. The two mandatory launch options are:
```

Comment:
Add two spaces after this line. Doing to will cause the following code block to appear on the following line.